services/minespace-web/src/components/common/DocumentTable.tsx

```diff
@@ -253,11 +280,26 @@ export const DocumentTable: FC<DocumentTableProps> = ({
   const bulkItems: MenuProps["items"] = [
     {
       key: "0",
+      icon: <DownloadOutlined />,
+      label: (
+        <button
+          type="button"
+          className="full add-permit-dropdown-button"
```

Comment:
is this classname even on minespace? I think I'd prefer for core-web to use "actions-dropdown-button" instead.

Reply:
Just redid the updates to this page.  Some stuff seemed to have snuck in that I didn't notice.